tools/vhost-user-video/v4l2_backend.h

```diff
@@ -15,8 +15,6 @@
 #include "standard-headers/linux/virtio_video.h"
 #include "virtio_video_helpers.h"
 
-#define ARRAY_SIZE(a) (sizeof(a) / sizeof((a)[0]))
-
```

Comment:
why you are removing this?

Reply:
It is declared in `#include "qemu/osdep.h"`, since I included this in the `udmabuf.c`, the compiler found a re-declaration.

And either way, it is better to avoid re-declarations, so I just included `qemu/osdep.h` in the source file.